lib/active_merchant/billing/gateways/card_stream.rb

```diff
@@ -283,6 +284,20 @@ def add_threeds_required(post, options)
         add_pair(post, :threeDSRequired, options[:threeds_required] || @threeds_required ? 'Y' : 'N')
       end
 
+      def add_global_threeds2_mpi_data(post, options)
```

Comment:
💬  Comment:

the method name doesn't need to point to the specific spreedly product.